icechunk-python/tests/test_zarr/test_properties.py

```diff
@@ -0,0 +1,51 @@
+import numpy as np
+import pytest
+from numpy.testing import assert_array_equal
+
+from icechunk import IcechunkStore, StorageConfig
+
+pytest.importorskip("hypothesis")
+
+import hypothesis.strategies as st
+from hypothesis import assume, given, settings
+
+from zarr.testing.strategies import arrays, numpy_arrays
```

Comment:
This is awesome... we should embrace this for the other tests as well in the future